layers/+lang/ess/packages.el

```diff
@@ -52,6 +52,8 @@
     :commands (R stata julia SAS)
     :init
     (progn
+      (spacemacs/register-repl 'ess-site 'julia)
+      (spacemacs/register-repl 'ess-site 'spacemacs/ess-start-repl "ess")
```

Comment:
`spacemacs/ess-start-repl` won't do what we want it to here. It only works if a buffer is already open in ESS mode. Instead each REPL provided by ESS should be registered separately, i.e.,

```emacs-lisp
      (spacemacs/register-repl 'ess-site 'julia)
      (spacemacs/register-repl 'ess-site 'R)
      (spacemacs/register-repl 'ess-site 'SAS)
      (spacemacs/register-repl 'ess-site 'stata)
```